tests/tr1/tests/cvt/sjis_0208/test.cpp

```diff
@@ -3,9 +3,97 @@
 
 #include "tdefs.h"
 #include <cvt/sjis_0208>
+#include <cvt/wstring>
+#include <iostream>
+#include <string>
+
+#define MYFILE  "sjis_0208"
+#define MYNAME  stdext::cvt::codecvt_sjis_0208<wchar_t>
+#define MYMAKE  mymake
+#define MYWCHAR wchar_t
+
+using namespace std;
+
+using Myistream  = basic_istream<MYWCHAR>;
+using Myostream  = basic_ostream<MYWCHAR>;
+using Mystring   = basic_string<MYWCHAR>;
+using Mysconvert = stdext::cvt::wstring_convert<MYNAME, MYWCHAR>;
+
+Mystring mymake() {
+    Mystring mystring;
+    Mysconvert myconv("");
+
+    // All ASCII and modified ASCII characters single byte characters
+    for (unsigned long ch = 0; ch <= 0x7f; ++ch) {

```

Comment:
Hm, I guess this introduces a ton of unnecessary `nul` characters to be tested but still works.